packages/contracts/contracts/L1/interfaces/IAssetManager.sol

```diff
@@ -9,12 +9,12 @@ interface IAssetManager {
     /**
      * @notice Represents the asset information of the vault of a validator.
      *
-     * @custom:field totalKro           Total amount of KRO in the vault.
+     * @custom:field totalKro           Total amount of KRO of delegators in the vault.
      * @custom:field totalKroShares     Total shares for KRO delegation in the vault.
      * @custom:field totalKgh           Total number of KGH in the vault.
      * @custom:field totalKroInKgh      Total amount of KRO which KGHs in the vault have.
      * @custom:field totalKghShares     Total shares for KGH delegation in the vault.
-     * @custom:field validatorKro       Amount of KRO that the validator self-delegated.
+     * @custom:field validatorKro       Amount of KRO that the validator deposited.
```

Comment:
Could you remove `self-delegate` also in the comments of `totalValidatorKro`, `ValidatorRegistered`, and `registerValidator`?